scripts/BTLH_data_exploration.R

```diff
@@ -14,9 +14,17 @@ raw_data <- read.csv("data/updated_raw_data.csv")
 
 # Capitalize all characters and factors across data frame 
 band_data <- mutate_all(raw_data, .funs=toupper)
+# EZ: Getting an error when I run this related to TailMid column
+# If you're also getting error, suggest replacing above with the following:
+# band_data <- raw_data %>%
+#   mutate(across(!TailMid, .funs = toupper))
 
```

Comment:
I'm not getting errors here. It is weird.

Reply:
Interesting!  Maybe we can try and figure this (and the other similar issue) out tomorrow.